list

```diff
@@ -689,6 +704,7 @@ learn.to
 lego.build
 lemde.fr
 letsharu.cc
+ley.su
```

Comment:
Didn't see any working samples out there.

Reply:
Links seem to expire fast, New ones can be created from homepage. Archive of original link (https://ley.su/?YOrX): https://archive.is/owzKS

New Link: https://ley.su/?RKZp
Target: `https://github.com/PeterDaveHello/url-shorteners`
New Archive: https://archive.is/1CSGa